flutter/lib/src/integrations/native_app_start_integration.dart

```diff
@@ -121,21 +134,36 @@ class NativeAppStartIntegration extends Integration<SentryFlutterOptions> {
 enum AppStartType { cold, warm }
 
 class AppStartInfo {
-  AppStartInfo(this.type,
-      {required this.start,
-      required this.end,
-      required this.pluginRegistration,
-      required this.mainIsolateStart});
+  AppStartInfo(
+    this.type, {
+    required this.start,
+    required this.pluginRegistration,
+    required this.mainIsolateStart,
+    DateTime? end,
```

Comment:
```suggestion
    this.end,
```

you can remove the `get` and `set`, and make it public like the other fields, just removing the `final` label